gm4_metallurgy/data/gm4_infinitas_shamir/advancements/water/place_mainhand.json

```diff
@@ -314,6 +314,7 @@
             "predicate": {
               "block": {
                 "blocks": [
+                  "minecraft:flower_pot",
                   "minecraft:water"
```

Comment:
Why is flower pot special? Should this not include all blocks that break when you place water inside?